src/playground/Weekdays/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { ToggleGroup, ToggleGroupItem } from "@/ui/toggleGroup"
+import { forwardRef } from "react"
+
+interface WeekdaysProps {
+  activatedDays?: string[]
+  daysOfTheWeek?: string[]
+}
+
+const DAYS_OF_THE_WEEK = [
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+  "Sunday",
+]
```

Comment:
I feel this should come as a parameter as it needs to be translated and it's also company-specific (start of the week could be different from company to company)

Reply:
Agreed. That's why this is a fallback in case you don't send your own days of the week.

Reply:
Hahaha I understand. Still, this is quite dangerous. I think we'll eventually have to add a `<LocalizationProvider />` in which we define things like:

* Error validation messages
* Default labels for obvious things: Submit buttons, tooltip aria-labels, etc
* Time things: Default week day, months, days, formatting, etc
* Maybe even money, etc?

So I would say that we can keep it like this while it's on the playground, but create that provider before we promote this to a full-fledged component.